libcaf_net/caf/detail/rfc6455.hpp

```diff
@@ -57,6 +57,10 @@ struct CAF_NET_EXPORT rfc6455 {
                              uint8_t flags = fin_flag);
 
   static ptrdiff_t decode_header(const_byte_span data, header& hdr);
+
+  static constexpr bool is_control_frame(uint8_t opcode) {
```

Comment:
The `noexcept` got lost from my last suggestion:

```suggestion
  static constexpr bool is_control_frame(uint8_t opcode) noexcept {
```

Reply:
Sorry about that. Should be fine now.